src/renderer/js/component/common/spinner.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import classnames from "classnames";
+
+export default ({ dark, className }) => {
+  return (
+    <div
+      className={classnames(
```

Comment:
@IGassmann @liamcardenas please be aware of this pattern/design for determining class names that @seanyesmunt has added.

Reply:
Interesting, why was it done this way? (no need to respond here, I'll ask about it tomorrow)